packages/ember-routing/lib/system/router.js

```diff
@@ -1563,4 +1564,9 @@ function representEmptyRoute(liveRoutes, defaultParentState, route) {
   }
 }
 
+deprecateProperty(EmberRouter.prototype, 'router', 'routerMicrolib', {
+  id: 'ember-router.router',
+  until: '2.16'
```

Comment:
I merged the deprecation PR so that the site was auto-published:

http://emberjs.com/deprecations/v2.x/#toc_ember-router-router-renamed-to-ember-router-routermicrolib